modules/rosa-sts-ocm-roles-and-permissions.adoc

```diff
@@ -0,0 +1,102 @@
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-user-role_{context}"]
+== User role
+
+The user role has no additional permissions. {cluster-manager} identifies this user role and creates trust between this role, the Red Hat account, and the two `ocm-role` resources that you create.
+
+//Used to identify the user by creating a trust between it, Red Hat account and the ocm role.
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-basic-role_{context}"]
+== Creating basic {cluster-manager} permissions
+
+The basic {cluster-manager} role permissions allow you to perform cluster maintenance within {cluster-manager}.
+
+.Procedure
+* Create a basic `ocm-role` resource:
++
+[source,terminal]
+----
+rosa create ocm-role
+----
++
+This command creates the role, prompting you for specifications for the role. The basic `ocm-role` resource contains the following permissions:
++
+.Associated permissions for the basic `ocm-role`
+[cols="1,2",options="header"]
+|===
+
+|Resource|Description
+
+| `iam:GetOpenIDConnectProvider`
+| This permission allows the basic role to retrieve information about the specified OpenID Connect (OIDC) provider.
+| `iam:GetRole`
+| This permission allows the basic role to retrieve any information for a specified role. Some of the data returned include the role's path, GUID, ARN, and the role's trust policy that grants permission to assume the role.
+| `iam:ListRoles`
+| This permission allows the basic role to list the roles within a path prefix.
+| `iam:ListRoleTags`
+| This permission allows the basic role to list the tags on a specified role.
+| `ec2:DescribeRegions`
+| This permission allows the basic role to return information about all of the enabled regions on your account.
+| `ec2:DescribeRouteTables`
+| This permission allows the basic role to return information about all of your route tables.
+| `ec2:DescribeSubnets`
+| This permission allows the basic role to return information about all of your subnets.
+| `ec2:DescribeVpcs`
+| This permission allows the basic role to return information about all of your virtual private clouds (VPCs).
+| `sts:AssumeRole`
+| This permission allows the basic role to retrieve temporary security credentials so that you you can use to access AWS resources that are beyond your normal permissions.
+| `sts:AssumeRoleWithWebIdentity`
+| This permission allows the basic role to retrieve temporary security credentials for users authenticated their account with a web identity provider.
+
+|===
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-admin-role__{context}"]
+== Creating admin {cluster-manager} permissions
+
+The admin {cluster-manager} role permissions include all of the permissions for basic {cluster-manager} permissions with some extra permissions required for administrative tasks. Some of these permissions allows you to create a role or attach policies to a role. You need this permissions role if you want to use the automatic creation mode for your ROSA Operator roles.
```

Comment:
You could move this description to a concept subsection similar to the "User role" one, about the OCM role. That could then also include the table that shows the permissions for the role. However it is structured, I think that there is enough content in this description along with the associated permissions tables to benefit from separating the concepts and the procedures.